vue-components/src/main.ts

```diff
@@ -1,7 +1,5 @@
-import HelloWorld from './components/HelloWorld.vue';
 import Button from './components/Button.vue';
 
 export {
 	Button,
-	HelloWorld,
```

Comment:
The dummy browser tests in #121 use `<HelloWorld>`.
We should think about which PR goes when into master so we don't create a big mess of failing CI etc :smile:

Reply:
I put #127 in front of it so this is no longer a problem =)